rotkehlchen/tests/unit/accounting/test_prefork_assets.py

```diff
@@ -14,6 +14,7 @@
 
 @pytest.mark.parametrize('mocked_price_queries', [prices])
 @pytest.mark.parametrize('ignored_assets', [[A_ETC], []])
+@pytest.mark.parametrize('db_settings', [{'include_fees_in_cost_basis': False}])
```

Comment:
Why making it false here and below?

Reply:
I made `include_fees_in_cost_basis` `False` in some tests to see that the previous approach with having fees as standalone events still works.